src/test/java/seedu/address/model/session/FeeTest.java

```diff
@@ -32,14 +32,19 @@ public void isValidFee() {
         assertFalse(Fee.isValidFee("10.230"));
         assertFalse(Fee.isValidFee("005"));
         assertFalse(Fee.isValidFee("005.20"));
+        assertFalse(Fee.isValidFee("0.00000001"));
+        assertFalse(Fee.isValidFee("10000"));
+        assertFalse(Fee.isValidFee("1000000000000000"));
+        assertFalse(Fee.isValidFee("-1000000000000000"));
+        assertFalse(Fee.isValidFee("-0"));
+        assertFalse(Fee.isValidFee("-100.40"));
```

Comment:
Probably we can do a 9999.999 check as well for 2dp?

Reply:
Good suggestion! Have added the check and it is false as expected.